CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.1.1
+
+Add `broker fix`
+
+`broker fix` tests your connection to FOSSA and your `git` integrations and gives you advice on how to fix any problems it encounters.
```

Comment:
Please provide or link to documentation here about:

- When would users use `broker fix`
- What can they expect `broker fix` to do specifically
- Give them a preview of what they should see

I think this could be added to the user guide, and then just linked to from here, if desired.
Also note that `CHANGELOG` is distributed with the binary, so make sure to use a fully qualified link if that's the route you take (I didn't realize this at the time I wrote the `v0.1.0` entry 😬)

Reply:
Done!